src/components/App.js

```diff
@@ -63,6 +64,9 @@ class App extends Component {
                 <Router path="/home">
                   <Home />
                 </Router>
+                <Router path="/Main">
```

Comment:
seach jyada discriptive hai is component ke liye istead of main

Reply:
You mean rename Main to Search, aisa?